build/images/base/Dockerfile

```diff
@@ -13,6 +13,25 @@
 # limitations under the License.
 
 ARG BUILD_TAG
+
+FROM ubuntu:22.04 AS gobgp-binary
```

Comment:
nit: could we move this intermediate build step _after_ the exiting one (cni binaries)